addons/web/static/tests/core/utils/sortable_tests.js

```diff
@@ -238,6 +238,46 @@ QUnit.module("Draggable", ({ beforeEach }) => {
         assert.verifySteps([]);
     });
 
+    QUnit.test(
+        "Drag has a default tolerance of 10 pixels before initiating the dragging",
+        async (assert) => {
+            assert.expect(3);
+
+            class List extends Component {
+                setup() {
+                    useSortable({
+                        ref: useRef("root"),
+                        elements: ".item",
+                        onDragStart() {
+                            assert.step("Initiation of the drag sequence");
+                        },
+                    });
+                }
+            }
+
+            List.template = xml`
+            <div t-ref="root" class="root">
+                <ul class="list">
+                    <li t-foreach="[1, 2, 3]" t-as="i" t-key="i" t-esc="i" class="item" />
+                </ul>
+            </div>`;
+
+            await mount(List, target);
+
+            // Move the element from only 5 pixels
+            const listItem = target.querySelector(".item:first-child");
+            await dragAndDrop(listItem, listItem, { x: listItem.getBoundingClientRect().width/2, y: listItem.getBoundingClientRect().height/2 + 5 });
```

Comment:
Here I had to specify that the drop occurs at the center of the element, since the mousedown is triggered at the center first. That's why it may seem more complicated than the 16.0 drop coordinates. But I did basically the same